types/src/council/index.ts

```diff
@@ -52,7 +53,7 @@ export type ISealedVote = {
 export class SealedVote
   extends JoyStructDecorated({
     voter: AccountId,
-    commitment: U8aFixed, // Hash
+    commitment: Hash,
```

Comment:
Why did we need to use a class constructor here and not the `Hash` interface from `@polkadot/types/interfaces` ?

Reply:
We cannot use an interface here, because interfaces are a compilation-time-only constructs in TypeScript.
Here we're passing values that are later passed to `Struct.with()`, so they have to be `Record<string, Constructor | string>`.
To `Struct.with()` we could just pass ''Hash'` (a string), but this will not work with `JoyStructDecorated` which needs `Constructor`s to do the "decorating"